astro/src/content/articles/authentication/sso-mfa-fusionauth.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Combinging SSO and MFA in FusionAuth
+description: In this article, you'll learn how to combine single sign-on with multi-factor authentication in FusionAuth.
+author: Matt Keib
+section: Authentication
+tags: single sign on mfa multi factor authentication sso security
+icon: /img/icons/sso-mfa.svg
```

Comment:
I think we normally use different icons that represent single sign-on and MFA. Can we try to match to those for consistency?